From 5dea152c7728f5a37370ad8a229115833e36b4f6 Mon Sep 17 00:00:00 2001
From: Rasmus Thomsen <oss@cogitri.dev>
Date: Sat, 7 Mar 2020 10:44:16 +0100
Subject: [PATCH] tests: use G_GSIZE_FORMAT instead of G_GUINT64_FORMAT for
 gsize

This fixes the build on 32-bit platforms with -Werror-format

fixes #32
---
 src/tests/allocs-by-size.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/allocs-by-size.c b/src/tests/allocs-by-size.c
index 666113a..6260d4b 100644
--- a/src/tests/allocs-by-size.c
+++ b/src/tests/allocs-by-size.c
@@ -109,7 +109,7 @@ allocs_by_size (SysprofCaptureReader *reader)
     {
       const Item *item = &g_array_index (ar, Item, i);
 
-      g_print ("%"G_GUINT64_FORMAT",%"G_GUINT64_FORMAT",%"G_GUINT64_FORMAT"\n",
+      g_print ("%"G_GSIZE_FORMAT",%"G_GSIZE_FORMAT",%"G_GSIZE_FORMAT"\n",
                item->size, item->cmp, item->count);
     }
 }
-- 
2.20.1

